data/companies.json

```
@@ -766,8 +766,24 @@
         "instagram": null,
         "linkedin": "https://www.linkedin.com/company/yco-pvt.-ltd./",
         "twitter": null,
-        "youtube": null,
+        "youtube": null
       }
     },   
+    {
+      "name": "BlackBox Technologies Pvt. Ltd.",
+      "establishedAt": 2018,
+      "location": "Kupondole, Lalitpur, Nepal",
```

To be consistent with the new changes, please update as follow
```suggestion
      "location": "Kupondole, Lalitpur",
```